packages/mdc-ripple/foundation.js

```diff
@@ -488,17 +485,20 @@ class MDCRippleFoundation extends MDCFoundation {
   /** @private */
   layoutInternal_() {
     this.frame_ = this.adapter_.computeBoundingRect();
-
     const maxDim = Math.max(this.frame_.height, this.frame_.width);
-    const surfaceDiameter = Math.sqrt(Math.pow(this.frame_.width, 2) + Math.pow(this.frame_.height, 2));
 
-    // 60% of the largest dimension of the surface
-    this.initialSize_ = maxDim * MDCRippleFoundation.numbers.INITIAL_ORIGIN_SCALE;
+    // Surface diameter is treated differently for unbounded vs. bounded ripples.
+    // Unbounded ripple diameter is calculated smaller since the surface is expected to already be padded appropriately
+    // to extend the pointer target, and the ripple is expected to meet the edges of the padded hitbox
```

Comment:
Maybe rephrase this to:

```
// to create a larger tap target, and ...
```

Reply:
I felt like saying "tap" was device-ist :P but I am considering just using the word hitbox instead... but that might be my inner gamer talking.